src/plugins/dataflow/nodes/dataflow-node-plot.tsx

```diff
@@ -114,5 +160,21 @@ function lineOptions() {
       }]
     },
   };
+
   return options;
 }
+
+
+enum MinOrMax {
+  Max,
+  Min
+}
+
+function returnMinOrMax (option: MinOrMax, node: any){
+  if (option === MinOrMax.Max){
+    return node.data.tickMax ?? node.data.dsMax;
+  }
+  if (option === MinOrMax.Min){
+    return  node.data.tickMin ?? node.data.dsMin;
+  }
+}
```

Comment:
I think it's a good idea to abstract this, since it was used in two places. The way this function is set up strikes me as overly complex, though. I think it would be fine to just make two functions, one for the min and one for the max.
```suggestion
const maxY = (node: any) => node.data.tickMax ?? node.data.dsMax;
const minY = (node: any) => node.data.tickMin ?? node.data.dsMin;
```
Then you'd have to change lines 34, 35, 122, and 123 as well.